batch_job/src/work_queue_factory.c

```diff
@@ -108,6 +107,10 @@ static int64_t factory_timeout = 0;
 
 struct batch_queue *queue = 0;
 
+static const char *mesos_master = NULL;
```

Comment:
I see. I noticed that there are many variables are defined as `static char *` and only used once in `main`, like (`amazon_credentials`, `amazon_ami`, `wrapper_command`, etc), is this a proper way of define variables?